doc/languages-frameworks/beam.section.md

```diff
@@ -56,11 +56,65 @@ Mix functions similarly to Rebar3, except we use `buildMix` instead of `buildReb
 
 Alternatively, we can use `buildHex` as a shortcut:
 
+#### buildMix - Elixir Phoenix example
+
+Here is how your default.nix file would look like
+
+```default.nix
+with import <nixpkgs-unstable> { };
```

Comment:
```suggestion
with import <nixpkgs> { };
```

once this hits stable channel, it shouldn't ask users to import unstable.

Reply:
Completely agree. Let's wait for it to hit stable channel. We wouldn't want the docs to be out of date in the meanwhile.

Reply:
How would you remember to fix it? Also if you read the unstable manual than we can assume that nixpkgs is unstable. Also choosing `nixpkgs-unstable` as an channel name is completely arbitrary.